pkg/dockercompose/convert.go

```diff
@@ -0,0 +1,313 @@
+package dockercompose
+
+import (
+	"fmt"
+
+	"github.com/kubernetes/kompose/pkg/kobject"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type ConvertBuild string
+
+const (
+	LOCAL        ConvertBuild = "local"
+	BUILD_CONFIG ConvertBuild = "build-config"
+	NONE         ConvertBuild = "none"
+)
+
+type KubernetesController string
+
+const (
+	DEPLOYMENT             KubernetesController = "deployment"
+	DAEMONSET              KubernetesController = "daemonSet"
+	REPLICATION_CONTROLLER KubernetesController = "replicationController"
```

Comment:
In what cases is a replicationController to be used? My understanding is that a replication controller is an older way to have the functionality provided by deployments (+ replicaSets).